GetCovid19Statistics/dcoder_block.yml

```diff
@@ -0,0 +1,14 @@
+id: COVID19_STATS
+fileName: index.js
```

Comment:
You need to build the block with @vercel/ncc module, follow the guide to build the block, it will create a single build file in dist/index.js

Once you have that, change here to dist/index.js